google-cloud-pub-sub-grpc/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/grpc/scaladsl/GooglePubSub.scala

```diff
@@ -94,21 +94,38 @@ object GooglePubSub {
       }
       .mapMaterializedValue(_.flatMap(identity)(ExecutionContexts.sameThreadExecutionContext))
 
+  /**
+   * Create a flow that accepts consumed message acknowledgements.
+   */
+  def acknowledgeFlow(): Flow[AcknowledgeRequest, AcknowledgeRequest, NotUsed] =
+    Flow
+      .setup { (mat, attr) =>
+        Flow[AcknowledgeRequest]
+          .mapAsync(1)(
+            req =>
+              subscriber(mat, attr).client
+                .acknowledge(req)
+                .map(_ => req)(mat.executionContext)
+          )
+      }
+      .mapMaterializedValue(_ => NotUsed)
+
   /**
    * Create a sink that accepts consumed message acknowledgements.
    *
    * The materialized value completes on stream completion.
    *
    * @param parallelism controls how many acknowledgements can be in-flight at any given time
    */
-  def acknowledge(parallelism: Int): Sink[AcknowledgeRequest, Future[Done]] =
+  def acknowledge(parallelism: Int): Sink[AcknowledgeRequest, Future[Done]] = {
```

Comment:
Should this method be deprecated like its javadsl counterpart?

Reply:
That was in the non-gRPC version.